tools.yml

```diff
@@ -2503,3 +2503,206 @@ toolshed.g2.bx.psu.edu/repos/earlhaminst/t_coffee/t_coffee/.*:
 #  env:
 #    OMP_NUM_THREADS: '{cores}'
 #    OMP_THREAD_LIMIT: '{cores}'
+toolshed.g2.bx.psu.edu/repos/galaxyp/openms_tmtanalyzer/TMTAnalyzer/.*:
+  cores: 4
+  mem: 8
+# Issues ↓:
+# - Exists in the database but resources are set differently.
+# toolshed.g2.bx.psu.edu/repos/iuc/anndata_inspect/anndata_inspect/.*:
+#   cores: 4  # (tool_destinations.yml)
+#   mem: 15.2  # (TPV database)
+toolshed.g2.bx.psu.edu/repos/bgruening/antismash/.*:
+  cores: 10
+  mem: 90
+  env:
+    _JAVA_OPTIONS: -XX:MaxPermSize=2G -Xmx{int(mem)}G -Xms1G
+# Issues ↓:
+# - Exists in the database but resources are set differently.
+# - Exists in the database under a different id. There original id:
+#   - Does NOT appear in the Tool Shed.
+#   - Does NOT appear in the Galaxy EU tool search.
+#   - Does NOT appear in usegalaxy.eu's PostgreSQL.
+# bam-to-wig: { mem: 20 }  (tool_destinations.yaml)
+# toolshed.g2.bx.psu.edu/repos/nilesh/rseqc/rseqc_bam2wig/.*:  # (tpv database)
+#   mem: 16
+# Issues ↓:
+# - Exists in the database but resources are set differently.
+# - Exists in the database under a different id. There original id:
+#   - Does NOT appear in the Tool Shed.
+#   - Does NOT appear in the Galaxy EU tool search.
+#   - Does appear in usegalaxy.eu's PostgreSQL.
+# bamCompare_deepTools: { mem: 10 } (tool_destinations.yaml)
```

Comment:
Those have all the same reason. They have been deployed locally before they have been released to the TS.

Since TPVs tool IDs are a bit restricted with its matching algorithm, I think we should just drop those tools here. They are not used anymore.
With the old tool ID matching both would have matched, the long and the short tool IDs.

Reply:
To be more concrete, please use the long form, and drop the short form.

Reply:
You could use a regex to match the short form id if needed.
```yaml
.*bamCompare_deepTools.*:
   cores: 10
```

Reply:
Thank @nuwang. I think there were some arguments against it, since short forms in regex mode can potentially also match other tools. Is there a way to specify the short-tool-id without regex and match in the same way Galaxy matches?

Reply:
I may be missing something, but the regex is matched against the full tool id. Is the issue is that a local tool has a full tool id like say `bamCompare_deepTools` instead of the full tool id of a typical toolshed tool like `toolshed.g2.bx.psu.edu/repos/bgruening/deeptools/bamCompare_deepTools/1.0.0`? In that case, it should be possible to match only the local one by using the regex `^bamCompare_deepTools$` which should match the start and end line markers for a very precise match. Or is the issue something else?

Reply:
Sounds legit, `^bamCompare_deepTools$` should work. Anyway, I removed tools like these due to the reasons pointed out by @bgruening.

Reply:
@nuwang Well now that I think of it, it has the disadvantage of not matching the same tool if Galaxy is using the long id in its database, right?

Reply:
A more complex regex using the `or` operator could provide an exact match for both, if you're concerned that a wildcard match might be too broad.
```
"^bamCompare_deepTools$|toolshed.g2.bx.psu.edu/repos/bgruening/deeptools/bamCompare_deepTools.*":
   cores: 2
```

Reply:
What was the initial reason to use regex by default? Wouldn't a match like Galaxy is doing be enough for most of the cases?